README.md

```diff
@@ -63,18 +87,18 @@ s3www requires access to view and list all files in the bucket.
 
 ```
 {
-	"Version": "2012-10-17",
+	"Version": "2012-10-17
 	"Statement": [
 		{
-			"Sid": "",
-			"Effect": "Allow",
-			"Action": "s3:GetObject",
+			"Sid": "
+			"Effect": "Allow
```

Comment:
This is a JSON document, how does it work if you remove `,` ?

Reply:
Sorry, issued with my sed command, I update my commit